src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -292,23 +293,61 @@ private function addWorkflowSection(ArrayNodeDefinition $rootNode)
                                         ->defaultNull()
                                     ->end()
                                     ->arrayNode('places')
+                                        ->beforeNormalization()
+                                            ->always()
+                                            ->then(function ($places) {
+                                                // It's an indexed array, BC Layer
```

Comment:
If this is only for BC, shouldn't add a deprecation warning?

Reply:
Hi. I just removed the BC break. Not we can support both format:
New:
```xml
            <framework:place name="start">
                <framework:metadata>
                    <framework:title>place start title</framework:title>
                </framework:metadata>
            </framework:place>
```
Old:
```xml
<framework:place>first</framework:place>
```

I'm not sure we really want to add (another) deprecation. Do you think it worth it?

Reply:
nope, looks like as is. Both make sense I think.

Reply:
So, the comment can be removed as this not a BC layer anymore, but just something normal